akka-stream-tests/src/test/scala/akka/stream/scaladsl/SourceSpec.scala

```diff
@@ -153,6 +153,29 @@ class SourceSpec extends AkkaSpec {
       gotten.toSet should ===(Set(0, 1, 2, 3, 4))
       out.expectComplete()
     }
+
```

Comment:
Had intention to add them (as well as docs and java api) after positive feedback on implementation approach.
Still if you need this to see API usage I add them right away.